.github/workflows/release-gh-pages.yml

```diff
@@ -23,7 +23,7 @@ jobs:
 
       - name: Build and push repositories json
         env:
-          GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_API_TOKEN: ${{ secrets.EG_GITHUB_TOKEN }}
```

Comment:
Shall we also use `EG_GITHUB_TOKEN` for `peaceiris/actions-gh-pages` below, for consistency in the entire workflow?

Reply:
we can unprotect that branch which would remove the need to pass this token to that action